client/src/legacy/LeftAndMain.js

```diff
@@ -1508,13 +1508,13 @@ $.entwine('ss', function($) {
   });
 });
 
-function escapeHTMLEntities(encodedString) {
-    const textArea = document.createElement('textarea');
-    textArea.innerHTML = encodedString;
-    return textArea.value;
+function decodeEntities(str) {
+    const ta = document.createElement('textarea');
+    ta.innerHTML = str;
```

Comment:
Please don't use short variable names like this - it might be obvious to you what `ta` means, but it's even _more_ obvious if it's just clearly named `textArea` (or `element` or some other meaningful name).